python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -95,9 +97,20 @@ class DiracBase(IBackend):
     _hidden = True
 
     def _setup_subjob_dataset(self, dataset):
+        """
+        Unknown what this ethod does other than return None:
+        Args:
+            Dataset (Dataset): This is a GangaDataset object, todo check this isn't a list
+        """
         return None
 
     def _setup_bulk_subjobs(self, dirac_ids, dirac_script):
+        """
+        This is the old bulk submit method which is used to construct the subjobs for a paraetric job
```

Comment:
What do you mean my the 'old' method? Is it not used any more? Also a typo in 'paraetric'.